201-vm-lb-winrm-windows/azuredeploy.json

```diff
@@ -0,0 +1,574 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "adminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "Admin username"
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Admin password"
+            }
+        },
+        "dnsLabelPrefix": {
+            "type": "string",
+			"metadata": {
+				"description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+			}
+        },
+        "vmNamePrefix": {
+            "type": "string",			
+            "defaultValue": "myVM",
+			"metadata": {
+				"description": "Virtual machine name prefix for primary vm instances"
+			}
+        },
+        "nvmNamePrefix": {
+            "type": "string",			
+            "defaultValue": "mytestVM",
+			"metadata": {
+				"description": "Virtual machine name prefix for secondary vm instances"
+			}
+        },
+        "lbName": {
```

Comment:
User should have a provision for providing his own friendly name. So, keeping it as parameter.